components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/AbstractGitpodPortForwardingService.kt

```diff
@@ -100,7 +99,15 @@ abstract class AbstractGitpodPortForwardingService : GitpodPortForwardingService
         return completableFuture
     }
 
+    private fun isPortExposingDisabled(): Boolean {
+        return System.getenv("GITPOD_DISABLE_JETBRAINS_LOCAL_PORT_EXPOSE")?.toBoolean() ?: false
+    }
+
     private fun syncPortsListWithClient(response: Status.PortsStatusResponse) {
+        if (isPortExposingDisabled()) {
```

Comment:
@mustard-mh I'm trying to understand the goal of the PR, could you help me please? 🙏 

We have local forwarded and exposed on Interent ports? For local forwarded ports some are forwarded by us other by JB itself, for instance via run configurations and native terminals. My undertand the goal is that:
- we need to disable forwarded ports regardless whether it is from us or JB? do we cover JB case?
- we don't need to disable exposed ports? Does this PR disable them as well? The view should not be completely empty right?

Reply:
> we need to disable forwarded ports regardless whether it is from us or JB? do we cover JB case?

@akosyakov Thanks for you review. Yes we do cover them all, I tested with current implements, JetBrains ports are covered as well, see screenshots below

|Enable|Disable|
|:-:|:-:|
|<img width="1132" alt="image" src="https://github.com/user-attachments/assets/2e8ea902-df28-4127-ada0-15e076f9cca9">|<img width="1072" alt="image" src="https://github.com/user-attachments/assets/fd1c1f21-5270-48ec-8920-afe8a77c33b0">|


> we don't need to disable exposed ports? Does this PR disable them as well?

Exposed ports are exposed by supervisor / ws-proxy , but not JetBrains IDEs. From JetBrains' UI, if you click the add button on the ports view, options all looks like port forwarding (`local -> remote`, or `remote -> local`)

So this view is not a good place for us to show exposed ports

I will write (prepare) a draft message for sales to communicate with users -> they may needs to use in workspace GP-CLI  i.e.`gp ports list` to list and process with ports after this changes

<img width="415" alt="image" src="https://github.com/user-attachments/assets/4e43784a-36f6-4997-a5a6-8495f4d2fb25">

Reply:
Btw, the naming is confusing, I'm going to change env name and function name from `local exposing` to `local forwarding`

Reply:
> Exposed ports are exposed by supervisor / ws-proxy , but not JetBrains IDEs. From JetBrains' UI, if you click the add button on the ports view, options all looks like port forwarding (local -> remote, or remote -> local)

It is true, but the backend plugin monitors add adds each port like local forwarded and exposed. We should keep exposed, only remove local forwarded.